### PR TITLE
Move infrastructure code into separate repository 

### LINKED ISSUE #1909: Move infrastructure code into separate repository
**Issue summary**

The `scripts/daily_snapshot`, `scripts/ruby_common`, and `scripts/sync_check` folders contain code for validating Forest and generating daily snapshots. This infrastructure code doesn't depend on Forest directly and should be moved to [forest-iac](https://github.com/ChainSafe/forest-iac) along with their workflows.

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes #1909

**Other information and links**

### CODE REVIEW

**Comment by lemmih** [comment]
Let's discuss in slack before you work more on this.

**Comment by lemmih** [comment]
Resolve the merge conflicts and also submit a new PR to `forest-iac` with the updated `dairly_snapshot.rb` code.

File: .github/workflows/scripts-lint.yml [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-jobs:
-  shellcheck:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Run shellcheck
-      uses: ludeeus/action-shellcheck@1.1.0
-      env:
-        SHELLCHECK_OPTS: --external-sources
```
Comment (LesnyRumcajs) [inline .github/workflows/scripts-lint.yml:20]: We should keep this check - Forest codebase not only relies on shell scripts for the infrastructure code but also for a check like here https://github.com/ChainSafe/forest/blob/main/scripts/add_license.sh. It doesn't cost much to run shellcheck, few seconds at most which is run in parallel anyway.

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]

File: .github/workflows/scripts-lint.yml (hunk 1 above)
Comment (tyshko5) [reply .github/workflows/scripts-lint.yml]: Rerturned

**Comment by LesnyRumcajs** [comment]
@tyshko5 not all files have been ported yet to the `forest-iac` repo, this should be first addressed before removing the ones here, or am I missing something?

image

**Comment by tyshko5** [comment]
> @tyshko5 not all files have been ported yet to the `forest-iac` repo, this should be first addressed before removing the ones here, or am I missing something?
> 
> image

Made a PR for that https://github.com/ChainSafe/forest-iac/pull/3

**Review by lemmih — APPROVED** [review]

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**